examples/load_contacts_and_show_some_layers.py

```diff
@@ -9,29 +9,35 @@
 import numpy as np
 import os
 
-def show_layers(popdict,show_ages=False):
 
+def show_layers(popdict, show_ages=False):
+    """
```

Comment:
So this is already in the code base of synthpops in the script contacts.py, all the way down at the end. I'd say if anything, let's remove the definition here and simply call sp.show_layers twice: once with show_ages = True, and once with it set to False. Good news: it also has the ability to show the contacts for a user specified number of people with parameter show_n.